kolibri/core/assets/src/vue/progress-bar/index.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+
+  <div>
+    <div class="visuallyhidden" id="progress-bar-label">{{ $tr('progressComplete') }}</div>
+    <div class="progress-bar-wrapper"
+      role="progress-bar"
```

Comment:
[Role should be `role="progressbar"`](https://www.w3.org/TR/wai-aria/roles#progressbar).